docs/server/tutorials/extensions.md

```diff
@@ -58,6 +58,32 @@ If your Sourcegraph instance is unable to connect to Sourcegraph.com (due to a f
 1. Run `src extensions publish -extension-id $USER/$NAME` in the clone directory to publish the extension locally to your Sourcegraph instance. Replace `$USER` with your Sourcegraph username and `$NAME` with with `codecov` or `basic-code-intel`.
 1. Enable the extension for your Sourcegraph user account by clicking the **Extensions** link in the top navigation, clicking **Add**, and selecting your username.
 
+### Private extension registry
+
+With Sourcegraph Enterprise, you can publish Sourcegraph extensions on your Sourcegraph instance to a private extension registry and control which extensions are available. Sourcegraph extensions that are published to the private extension registry on your instance are only visible to other users on your instance.
+
+#### Inheritance of Sourcegraph extensions from Sourcegraph.com
+
+Branded Sourcegraph instances inherit extensions from Sourcegraph.com with [`extensions.remoteRegistry`](https://about.sourcegraph.com/docs/config/site/#remoteregistry) set to `"https://sourcegraph.com/.api/registry"`. The OSS version of Sourcegraph has no dependencies on external services, and its `extensions.remoteRegistry` is set to `""`.
+
+You can disable inheritance by setting [`extensions.remoteRegistry`](https://about.sourcegraph.com/docs/config/site/#remoteregistry) to `false` in your site configuration:
```

Comment:
Tri-state (and multi-type) configuration options like this don't seem that intuitive to me (e.g. empty string vs false). Did you consider alternatives?